tests/storage/snapshots/conftest.py

```
@@ -149,3 +173,214 @@ def vm_restore_with_predictable_names(
+            size=BLANK_DV_SIZE,
+            storage_class=storage_class_name,
+            api_name="storage",
+        )
+        blank_dv.to_dict()
+        blank_dv.res["metadata"].pop("namespace", None)
+        add_dv_to_vm(vm=vm, template_dv=blank_dv.res)
+    running_vm(vm=vm)
+    return vm
+
+
+@pytest.fixture()
```

Ruff (0.16.1)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)